src/main/java/org/apache/commons/collections4/bloomfilters/BloomFilter.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.commons.collections4.bloomfilters;
+
+import java.util.BitSet;
+
+/**
+ * The interface for all BloomFilter implementations. Instances of BloomFilters
+ * should be immutable.
+ */
+public interface BloomFilter {
+
+    /**
+     * Return true if other &amp; this == other
```

Comment:
Start Javadoc comments with "Returns" instead of "Return"; "Gets" instead of "Get" and so on.